peps/pep-0705.rst

```diff
@@ -368,18 +395,18 @@ There are no known security consequences arising from this PEP.
 How to teach this
 =================
 
-Suggestion for changes to the :mod:`typing` module, in line with current practice:
+Suggested changes to the :mod:`typing` module documentation, in line with current practice:
 
 * Add this PEP to the others listed.
 * Add ``typing.ReadOnly``, linked to TypedDict and this PEP.
 * Add the following text to the TypedDict entry:
 
-Individual items can be excluded from mutate operations using ReadOnly, allowing them to be read but not changed. This is useful when the exact type of the value is not known yet, and so modifying it would break structural subtypes. *insert example*
+The ReadOnly type qualifier indicates that an item declared in a TypedDict definition may be read but not mutated (added, modified or removed). This is useful when the exact type of the value is not known yet, and so modifying it would break structural subtypes. *insert example*
```

Comment:
```suggestion
The ``ReadOnly`` type qualifier indicates that an item declared in a ``TypedDict`` definition may be read but not mutated (added, modified or removed). This is useful when the exact type of the value is not known yet, and so modifying it would break structural subtypes. *insert example*
```